src/PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp

```diff
@@ -4,11 +4,28 @@
 #pragma once
 
 #include <cstddef>
+#include <memory>
+#include <string>
 
 #include "DataStructures/DataBox/Tag.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/Serialize.hpp"
 #include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "Utilities/TMPL.hpp"
 
 namespace Elasticity {
+
+namespace OptionTags {
+template <size_t Dim>
+struct ConstitutiveRelation : db::SimpleTag {
```

Comment:
Please document this option struct

Reply:
I think currently almost none of the option tags have a docstring, because they have a `help` string instead.